core-bundle/src/Command/UserPasswordCommand.php

```diff
@@ -144,22 +152,21 @@ private function askForPassword($label, InputInterface $input, OutputInterface $
     /**
      * Validates the password length and creates the password hash.
      *
-     * @param string $password
+     * @param string   $password
+     * @param int|null $minLength
      *
      * @throws InvalidArgumentException
      *
      * @return string
      */
-    private function validateAndHashPassword($password)
+    private function validateAndHashPassword($password, $minLength)
     {
-        $this->framework->initialize();
-
-        /** @var Config $config */
-        $config = $this->framework->getAdapter(Config::class);
-        $passwordLength = $config->get('minPasswordLength') ?: 8;
+        if (null === $minLength) {
+            $minLength = 8;
+        }
```

Comment:
```suggestion
```

Reply:
Sorry, I thought that was your whole point. It's changed again, now.